### ISSUE TITLE
Esc doesn't work when gear icon has focus

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#107665]
Testing #107610

* ubuntu20 w/ orca
* tab into the gear icon, e.g so that it has a focus border
* :bug: press `Esc` and nothing happens, press `Tab` and you end up in the activity bar

**Comment by roblourens** [comment]
I see that the action bar intercepts all "escape" keypresses, @isidorn it looks like you worked with this, do you know of a way to disable or intercept this or should I just plumb an option to disable that behavior through from ToolBar?

**Comment by isidorn** [comment]
@roblourens The thing I changed: previosuly pressing `esc` in actionBar would lose focus. I just changed that pressing `esc` should have no immediate effect. It just triggers the [`onDidCancel`](https://github.com/microsoft/vscode/blob/1287f6b14a4c488549ed01f53af11e61b9d5ac73/src/vs/base/browser/ui/actionbar/actionbar.ts#L68) event. So ideally you should be able to listen to this event and do what you want.

**Comment by roblourens** [comment]
I want to be able to trigger a command with escape in the normal way, so I want actionBar to not intercept escape at all. I'll add an option or that
